cvprac/cvp_client.py

```diff
@@ -111,7 +111,8 @@ class CvpClient(object):
     # CVP node.
     NUM_RETRY_REQUESTS = 3
 
-    def __init__(self, logger='cvprac', syslog=False, filename=None):
+    def __init__(self, logger='cvprac', syslog=False, filename=None,
+                 log_level='INFO'):
```

Comment:
I'm fine with leaving it in both places or removing it from init. I put it in both places because that is how the issue was worded.